Makefile

```diff
@@ -127,17 +134,19 @@ lint: golangci-lint ## Run golangci-lint against code.
 
 ##@ Build
 
-.PHONY: manager
-manager: ## Build manager binary.
-	go build -o $@
+manager: $(shell find -name "*.go") go.mod go.sum  ## Build manager binary.
+	go build -o $@ ./cmd/manager
+
+manager-hub: $(shell find -name "*.go") go.mod go.sum  ## Build manager-hub binary.
+	go build -o $@ ./cmd/manager-hub
 
 .PHONY: run
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
 .PHONY: docker-build
-docker-build: unit-test ## Build docker image with the manager.
-	docker build -t $(IMG) .
+docker-build: ## Build docker image with the manager.
+	docker build -t $(IMG) --build-arg TARGET=manager .
```

Comment:
is there a target for building the manager-hub image?

Reply:
Not yet, I did not want to add too many targets. Do you think there should be one?

Reply:
i think yes. We are all using docker-build target to build images, so it is better to have one for hub also. Maybe define 2 targets:
docker-build-manager
docker-build-hub

Reply:
PTAL